docs/KubernetesInstallation.md

```diff
@@ -0,0 +1,38 @@
+# Installing GKE plugin on Jenkins running in Kubernetes
+This document provides guidance on how to configure Jenkins running within Kubernetes in order to meet the plugin's pre-requisites, namely that the `kubectl` binary be installed on the agent container. 
+
+## Installing on an existing Jenkins installation with Kubernetes Cloud
+This section assumes the user has an existing Jenkins installation configured to use a Kubernetes cluster as the cloud via the [kubernetes-plugin](https://github.com/jenkinsci/kubernetes-plugin). Furthermore, it's assumed that the GKE plugin has been installed either through the [Plugin Manager](https://jenkins.io/doc/book/managing/plugins/) or else using the [SourceBuildInstallation](SourceBuildInstallation.md) guide. This content provides guidance on changing the existing Kubernetes cloud configuration to be compatible with the GKE plugin.
+
+*  Within your Jenkins installation's web console, navigate to "Manage Jenkins" > "Configure System" > "Cloud" > "Images" > "Container"
+
+*  Modify the "Docker image" field to: `gcr.io/jenkins-gke-plugin/jenkinsagent`
```

Comment:
Does a note need to be made for the image tag?

Reply:
The cloud config for the k8s plugin does not appear to provide a field for that.